HRTool/src/main/webapp/index.html

```diff
@@ -1,87 +1,82 @@
 <!DOCTYPE html>
-<html>
+<html ng-app="app">

```

Comment:
The index page must not contain any logic.
The only allowed element inside the body is <ui-view></ui-view> (+ a bunch of imports).
Imports always go inside the body, below ui-view (unless required otherwise)
Use CDN for library imports. No libraries should be included with the project.